File: Python Basics/Pre_Exam-20-04-2024/02. Maiden Party/02.Maiden Party.py
budget = float(input())
love_message_count = int(input())
wax_roses_count = int(input())
keychains_count = int(input())
caricature_count = int(input())
good_luck_count = int(input())

total_goods = love_message_count + wax_roses_count + keychains_count + caricature_count + good_luck_count

saved_money_1 = love_message_count * 0.6 + wax_roses_count * 7.2 + keychains_count * 3.6 + caricature_count * 18.2 + good_luck_count * 22

if total_goods >= 25:
    saved_money_1 *= 0.65

final_saved_money = saved_money_1 * 0.9

difference = abs(final_saved_money - budget)

if final_saved_money > budget:
    print(f"Yes! {difference:.2f} lv left.")
else:
    print(f"Not enough money! {difference:.2f} lv needed.")

File: CSharp Development/C# Introduction - Jan 2026/16. Exam/02.MaidenParty.cs
using System;

double partyPrice = double.Parse(Console.ReadLine()!);
int loveMessagesCount = int.Parse(Console.ReadLine()!);
int waxRosesCount = int.Parse(Console.ReadLine()!);
int keychainsCount = int.Parse(Console.ReadLine()!);
int caricaturesCount = int.Parse(Console.ReadLine()!);
int surpriseCount = int.Parse(Console.ReadLine()!);

double totalRevenue =
    loveMessagesCount * 0.60 +
    waxRosesCount * 7.20 +
    keychainsCount * 3.60 +
    caricaturesCount * 18.20 +
    surpriseCount * 22.00;

int totalItemsCount =
    loveMessagesCount +
    waxRosesCount +
    keychainsCount +
    caricaturesCount +
    surpriseCount;

if (totalItemsCount >= 25)
{
    totalRevenue *= 0.65; // 35% discount
}

double profit = totalRevenue * 0.90; // 10% hosting costs

if (profit >= partyPrice)
{
    double moneyLeft = profit - partyPrice;
    Console.WriteLine($"Yes! {moneyLeft:F2} lv left.");
}
else
{
    double neededMoney = partyPrice - profit;
    Console.WriteLine($"Not enough money! {neededMoney:F2} lv needed.");
}
